src/databricks/labs/remorph/connections/database_manager.py

```diff
@@ -8,6 +8,8 @@
 from sqlalchemy import text
 from sqlalchemy.exc import OperationalError
 
+from snowflake.sqlalchemy import URL as SnowflakeURL
+
```

Comment:
I think I'd prefer to move this into just the scope where it's used, can we do this?

The linter will probably complain, but given that this seems to be a fairly heavyweight import I think I'd prefer to cheat the linter.

Reply:
Great catch. I moved the import to the connector scope in the latest commit.